From df2a75f4864b30011ab6a6f365d9378d8eafa53b Mon Sep 17 00:00:00 2001
From: loqs <loqs@users.noreply.github.com>
Date: Mon, 20 Nov 2017 23:44:07 +0000
Subject: [PATCH] Add elevator aliases for bfq-mq and mq-deadline based upon
 https://git.kernel.org/pub/scm/linux/kernel/git/axboe/linux-block.git/commit/block/elevator.c?id=8ac0d9a81edf2ef4a2268b65b802a6b856dc77e6
 add bfq as an alias for bfq-sq and deadline for mq-deadline

---
 block/bfq-sq-iosched.c | 1 +
 block/mq-deadline.c    | 1 +
 2 files changed, 2 insertions(+)

diff --git a/block/bfq-sq-iosched.c b/block/bfq-sq-iosched.c
index 4bbd7f4c0154a..07776ca37716d 100644
--- a/block/bfq-sq-iosched.c
+++ b/block/bfq-sq-iosched.c
@@ -5395,6 +5395,7 @@ static struct elevator_type iosched_bfq = {
 	.icq_align =		__alignof__(struct bfq_io_cq),
 	.elevator_attrs =	bfq_attrs,
 	.elevator_name =	"bfq-sq",
+	.elevator_alias =	"bfq",
 	.elevator_owner =	THIS_MODULE,
 };
 
diff --git a/block/mq-deadline.c b/block/mq-deadline.c
index a1cad4331eddc..0179e484ec98c 100644
--- a/block/mq-deadline.c
+++ b/block/mq-deadline.c
@@ -657,6 +657,7 @@ static struct elevator_type mq_deadline = {
 #endif
 	.elevator_attrs = deadline_attrs,
 	.elevator_name = "mq-deadline",
+	.elevator_alias = "deadline",
 	.elevator_owner = THIS_MODULE,
 };
 MODULE_ALIAS("mq-deadline-iosched");
